tests/performance/jshint.js

```diff
@@ -1111,7 +1111,7 @@ var state    = require("./state.js").state;
 var style    = require("./style.js");
 
 // We need this module here because environments such as IE and Rhino
-// don't necessarilly expose the 'console' API and browserify uses
+// don't necessarily expose the 'console' API and browserify uses
```

Comment:
I don't think there are any downsides to fixing this, though I want to check with other teammates since I'm assuming this code was copied from an existing repo somewhere.

Reply:
This is ok